frontend/selectors/addressBook.spec.js

```diff
@@ -1,6 +1,6 @@
 import * as selectors from './addressBook';
 
-const statePrefix = '../UserReducers';
+const statePrefix = '@shopgate/user/UserReducers';
```

Comment:
Using path aliases is not possible in extensions at the moment. It will fail in the deployment process.